src/components/Modal/ModalPortal.js

```diff
@@ -36,6 +37,7 @@ class ModalPortal extends React.Component {
 ModalPortal.propTypes = {
   className: PropTypes.string,
   children: PropTypes.node.isRequired,
+  zIndex: PropTypes.number,
```

Comment:
it's required in PopperTooltip but it should not be required here

Reply:
`PopperTooltip` is not the only component which uses ModalPortal

Reply:
The `zIndex` prop itself for `ModalPortal` is not required (can be omitted). But if what you mean is that there should be no prop `zIndex` for `ModalPortal` at all, then I need your advice on how to provide z-index to the overlay.
My case is that when I open UserGuideTooltip, the black overlay gets covered by any background element that has z-index (LiveChat navigation menu for example). I need to provide z-index to the overlay somehow.
My case is that when I

Reply:
I don't know why but I assumed that you are requiring it here. 🤔The way it is now is fine.